Update the documentation to match 3.6 release (#1223)
scylladb/scylla-monitoring: docs/source/monitoring_stack.rst

.. code-block:: sh

   wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-3.6.0.tar.gz

>    tar -xvf scylla-monitoring-3.6.0.tar.gz

Reviewer: Question: can you create (and update) a 3.6 tag with hold the latest 3.6? 3.6.0 now, 3.6.1 later.
With this, you do not have to change this with each patch release.

Author: This is what #1039  is about, I think @lauranovich and @dgarcia360 will take care of it

Reviewer: Not the same
I'm asking if you can release 3.6 so it will have the latest release zip
Not doc related

Author: do you mean that the gz file will be called latest?
The issue was about taking the 3.6 in the name from somewhere else, so it will be updated without me updating it manually

   ./start-all.sh -l -d prometheus-data

> Configure rsyslog on Scylla
> ...........................

Reviewer: ```suggestion
Configure rsyslog on each Scylla node
.....................................
```

> Scylla Monitoring generates metrics and alerts from logs. To get full functionality, you should use rsyslog_. Scylla Monitoring will act as an additional rsyslog server.

Reviewer: ```suggestion
Scylla Monitoring Stack can collect Scylla logs with Loki, and generates metrics and alerts base on these logs. 
To use this feature, you need to direct logs from each Scylla node to Loki.
A recommended way to do that is using rsyslog_, where Scylla Monitoring (Loki) acts as an additional rsyslog server.
Note that you can have Scylla to send logs to more than one log collection service.
```

.. _rsyslog: https://www.rsyslog.com/

**Prerequisite**, make sure rsyslog is installed and running. If rsyslog is not installed, follow the installation instruction_.

.. _instruction: https://www.rsyslog.com/doc/v8-stable/installation/index.html

Add scylla's rsyslog configuration file. Add the file: ``/etc/rsyslog.d/scylla.conf``.

If Scylla monitoring IP is 10.0.0.1, the file should look like

.. code-block:: sh

   if $programname ==  'scylla' then @@10.0.0.1:1514;RSYSLOG_SyslogProtocol23Format

> Restart rsyslog for the configuration to take effect.

Reviewer: How?
Better add a command

   ./start-all.sh -l -d prometheus-data

Configure rsyslog on each Scylla node
.....................................

> generates metrics and alerts from logs. To get full functionality, you should use rsyslog_. Scylla Monitoring will act as an additional rsyslog server.

Reviewer 2: what generates the alerts rsyslog, or Scylla monitoring, or the Scylla nodes?

> Scylla Monitoring Stack can collect Scylla logs with Loki, and generates metrics and alerts base on these logs. 

Reviewer 2: ```suggestion
Scylla Monitoring Stack collects Scylla logs using Loki and generates metrics and alerts based on these logs. 
```

To use this feature, you need to direct logs from each Scylla node to Loki.

> A recommended way to do that is using rsyslog_, where Scylla Monitoring (Loki) acts as an additional rsyslog server.

Reviewer 2: ```suggestion
The recommended method to do this is by using rsyslog_, where Scylla Monitoring (Loki) acts as an additional rsyslog server.
```

> Note that you can have Scylla to send logs to more than one log collection service.

Reviewer 2: ```suggestion
.. note:: Scylla can send logs to more than one log collection service.
```

Reviewer 2: HOW?? - we should link to where this is done

Author: why?

Restart rsyslog for the configuration to take effect.

.. code-block:: sh

>    systemctl restart rsyslog

Reviewer 2: why restart and not start?

Author: See the first paragraph of this doc, it should be working already
